src/python/BuildingControlsSimulator/DataClients/DataClient.py

```diff
@@ -307,6 +307,21 @@ def get_data(self):
             simulation_epw_dir=self.simulation_epw_dir,
         )
 
+        #add nsrdb solar data fields
+        self.weather.fill_nsrdb_data = self.weather.get_nsrdb(
+            self.sim_config["latitude"], self.sim_config["longitude"], self.sim_config["start_utc"].year
+        )
+        if not self.weather.fill_nsrdb_data.empty:
```

Comment:
this logic should be added into `weather.get_nsrdb()`. To the extent possible we don't want to complicate `DataClient.get_data()` with implementation details of the channels.

Reply:
sure, makes sense.